--- /etc/lightdm/lightdm.conf	2014-11-04 21:01:16.000000000 +0000
+++ /etc/lightdm/lightdm.conf-zonamadev	2015-12-23 12:40:57.282097266 +0000
@@ -81,8 +81,8 @@
 [SeatDefaults]
 #type=xlocal
 #xdg-seat=seat0
-#pam-service=lightdm
-#pam-autologin-service=lightdm-autologin
+pam-service=lightdm
+pam-autologin-service=lightdm-autologin
 #pam-greeter-service=lightdm-greeter
 #xserver-command=X
 #xserver-layout=
@@ -102,7 +102,7 @@
 #greeter-show-manual-login=false
 #greeter-show-remote-login=true
 #user-session=default
-#allow-user-switching=true
+allow-user-switching=false
 #allow-guest=true
 #guest-session=
 #session-wrapper=lightdm-session
@@ -114,8 +114,8 @@
 #session-setup-script=
 #session-cleanup-script=
 #autologin-guest=false
-#autologin-user=
-#autologin-user-timeout=0
+autologin-user=vagrant
+autologin-user-timeout=0
 #autologin-in-background=false
 #autologin-session=UNIMPLEMENTED
 #exit-on-failure=false
